src/configimpl.l

```diff
@@ -2068,6 +2059,23 @@ void Config::checkAndCorrect(bool quiet, const bool check)
 
 }
 
+static void append_delemited(QCString& a, const char * name, ConfigObsolete* value)
```

Comment:
delemited -> delimited I assume, it is not quite clear what this function does from the name (`appendAttribute` sounds more appropriate). Also, `QCString` has a `quoted()` member now, so I think this can be used instead of `std::quoted()`.